content/en/docs/concepts/services-networking/ingress.md

```diff
@@ -74,7 +74,7 @@ A minimal Ingress resource example:
 
 {{< codenew file="service/networking/minimal-ingress.yaml" >}}
 
-As with all other Kubernetes resources, an Ingress needs `apiVersion`, `kind`, and `metadata` fields.
+As with all other Kubernetes resources, an Ingress needs `apiVersion`, `kind`, `metadata` and `spec` fields.
```

Comment:
No. The `spec` field is pretty common among k8s resources, but there are still some resources that do not have a spec, for example , `Secret`.

Reply:
But it is required for Ingress.
Would this below not be more helpful?

```suggestion
An Ingress needs `apiVersion`, `kind`, `metadata` and `spec` fields.
```